sdk/core/core-arm/lib/lroPollStrategy.ts

```diff
@@ -99,9 +99,9 @@ export abstract class LROPollStrategy {
     error.request = stripRequest(this._pollState.mostRecentRequest);
     error.response = this._pollState.mostRecentResponse;
     error.message = `Long running operation failed with status: "${this._pollState.state}".`;
-    error.body = this._pollState.resource;
-    if (error.body) {
-      const innerError: any = error.body.error;
+    error.response.parsedBody = this._pollState.resource;
```

Comment:
Should we use the `error.details` here instead?
`this._pollState.resource` is not really the parsed body of the response here right?

@sadasant, @chradek, @bterlson

Reply:
`details` seems appropriate to me.

Reply:
On second thoughts `this._pollState.resource` has already been updated to parsed response body elsewhere, so this is ok for now, but needs deeper investigation. Opened issue #5680